src/actions/deploy.ts

```diff
@@ -56,7 +56,9 @@ export async function deploy(
    for (const ingressResource of ingressResources) {
       await kubectl.getResource(
          KubernetesConstants.DiscoveryAndLoadBalancerResource.INGRESS,
-         ingressResource.name
+         ingressResource.name,
```

Comment:
not sure if this is out of scope for this PR, but wonder if we can standardize K8s objects to have namespace in the same field (ie metadata.namespace)? feel like having all objects look the same would make it easier to work with them

Reply:
There's a difference between the actual k8s object and what our app passes around / parses. Standardizing is good but the Resource type is something our code transforms the actual k8s object into so it will have a different model. The Resource type is for our application and our application doesn't need to know about the concept of metadata vs something else.

If we make Resource type the same as a k8s object then a developer would have to know what's useful for this application vs what's not. That's a lot harder than just using a simple object that's parsed from a more complicated one.

Reply:
makes sense!